### PR TITLE
Google Cloud Translate operator

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

File: airflow/contrib/operators/gcp_translate_operator.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                target_language=self.target_language,
+                format_=self.format_,
+                source_language=self.source_language,
+                customization_ids=self.customization_ids,
+                model=self.model,
+            )
+            self.log.debug("Translation %s", translation)
+            return translation
+        except ValueError as e:
+            self.log.error('An error has been thrown from translate method:', e)
```
Comment (sprzedwojski) [inline airflow/contrib/operators/gcp_translate_operator.py]:
```suggestion
            self.log.error('An error has been thrown from translate method:\n', e)
```
Maybe we could print the error details in a new line?

File: docs/howto/operator.rst [hunk 2]
```diff
@@ -1735,3 +1735,52 @@ More information
...
+
+Basic usage of the operator:
+
+.. literalinclude:: ../../airflow/contrib/example_dags/example_gcp_translate.py
+      :language: python
+      :dedent: 4
+      :start-after: [START howto_operator_translate_text]
+      :end-before: [END howto_operator_translate_text]
+
+The result of translation is available ad dictionary or array of dictionaries accessible via the usual
```
Comment (sprzedwojski) [inline docs/howto/operator.rst]:
```suggestion
The result of translation is available as dictionary or array of dictionaries accessible via the usual
```

File: tests/contrib/hooks/test_gcp_translate_hook.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        result = self.hook.translate(
+            values=['zażółć gęślą jaźń'],
+            target_language='en',
+            format_='text',
+            source_language=None,
+            customization_ids=['aa'],
+            model='base',
+        )
+        # Then
+        # ProductSet ID was provided explicitly in the method call above, should be returned from the method
```
Comment (sprzedwojski) [inline tests/contrib/hooks/test_gcp_translate_hook.py]:
```suggestion
```

File: airflow/contrib/operators/gcp_translate_operator.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+    * ``detectedSourceLanguage``: The detected language (as an
+      ISO 639-1 language code) of the text.
+    * ``translatedText``: The translation of the text into the
+      target language.
+    * ``input``: The corresponding input value.
+    * ``model``: The model used to translate the text.
+
+    If only a single value is passed, then only a single
+    dictionary is set as XCom return value
```
[the remark this answers is not in the document]
Comment (potiuk) [reply airflow/contrib/operators/gcp_translate_operator.py]: > Fixed.

File: airflow/contrib/operators/gcp_translate_operator.py (hunk 1 above)
Comment (potiuk) [reply airflow/contrib/operators/gcp_translate_operator.py]: Why not. fixed :)

File: docs/howto/operator.rst (hunk 2 above)
Comment (potiuk) [reply docs/howto/operator.rst]: 👍

File: tests/contrib/hooks/test_gcp_translate_hook.py (hunk 3 above)
Comment (potiuk) [reply tests/contrib/hooks/test_gcp_translate_hook.py]: oups

**Comment by potiuk** [comment]
All Fixed. Closing this one and opening one to Apache repo.

**Pull request closed without merging**